### PR TITLE
added Getting Started documentation for python client library

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: python/documentation.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+The package `crate` contains both the Python Database API implementation and the Crate dialect for SQLAlchemy.
+
+In this example we only cover the usage of the plain Crate client.
+
+## Usage
+
+### Connecting to Crate Server
+
+It's very simple to connect to connect to a single Crate node or Crate cluster. You need to import the `connect` method from the `crate.client` module and provide the `<host>:<port>` of a single Crate instance or a list of them to connect to. The port is the `HTTP` port (default `4200`) of the Crate node.
```
Comment (joemoe) [inline python/documentation.md]: `to connect to connect` is one `to connect` to much.

File: python/documentation.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+```
+
+### Closing the connection
+
+Both the connection and the cursor can be closed on demand, rather then whenever `.__del__()` is called. Cursors and connections that have been closed cannot be used any more and will raise an error if any operation is attempted.
+
+```python
+cursor.close()
+connection.close()
+```
```
Comment (joemoe) [inline python/documentation.md]: i think we should also write a paragraph about blobs as soon as we have to blob stuff implemented

**Comment by joemoe** [comment]
i think this is good. i'm not sure how we should separate documentation of client and documentation of sample app. to me this could be one the same page. therefore i would combine the readme with the documentation.

File: python/documentation.md (hunk 2 above)
Comment (chaudum) [inline python/documentation.md]: you're right, we should also use the python blob api 
https://crate-python.readthedocs.org/en/latest/blobs.html
will update the python app and the docs

**Comment by joemoe** [comment]
ack

**Comment by kovrus** [comment]
ack

**Pull request merged**